test_x_scraping.py

```
@@ -0,0 +1,528 @@
+#!/usr/bin/env python3
+"""
+Comprehensive test suite for X (Twitter) scraping functionality.
+Tests the apify_handler module with proper mocking to avoid API dependencies.
+"""
+
+import pytest
+import asyncio
```

Ruff (0.11.9)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)